pom.xml

```diff
@@ -27,7 +27,7 @@
 
         <!-- Keep these dependency versions in sync with those pulled in by beam;
              check https://mvnrepository.com/artifact/org.apache.beam -->
-        <google-cloud.version>1.49.0</google-cloud.version>
+        <google-cloud.version>1.61.0</google-cloud.version>
```

Comment:
I'm surprised this doesn't throw all sorts of errors. I think the ingestion-beam tests are being skipped, but this change could have significant impact on ingestion-beam.

Reply:
additional info: beam 2.13.0 depends on google-cloud-core `1.61.0`, so this is picking up bug fixes for ingestion-sink, but it's also putting the dependency back in sync with beam.

Reply:
I've updated the circleci config to run tests if the root pom.xml changed and there's a child pom.xml in the working dir